internal/web_server/web/handlers/api/admin_api.go

```diff
@@ -815,7 +821,7 @@ func (aah *AdminApiHandler) editQuestion(c echo.Context) error {
 		Alternatives:     alternatives,
 	}
 
-	question, err := createQuestion(c, questionForm, aah.sharedData.DB)
+	question, err := createOrEditQuestion(c, questionForm, aah.sharedData.DB)
```

Comment:
ngl its kinda confusing that you reuse the same endpoint for creation and editing of a question. I do realize it's been this way earlier, so i think its fine. something to watch out for i guess